connector/src/main/java/org/geysermc/connector/network/translators/collision/translators/SnowCollision.java

```diff
@@ -90,6 +92,11 @@ public boolean correctPosition(GeyserSession session, BoundingBox playerCollisio
             double boxMaxY = (boundingBoxes[0].getMiddleY() + y) + (boundingBoxes[0].getSizeY() / 2);
             // If the player actually can't step onto it (they can step onto it from other snow layers)
             if ((boxMaxY - playerMinY) > 0.5) {
+                // Make the player jump over the snow, which makes walking seamless
+                SetEntityMotionPacket motionPacket = new SetEntityMotionPacket();
+                motionPacket.setRuntimeEntityId(session.getPlayerEntity().getGeyserId());
+                motionPacket.setMotion(Vector3f.from(0, 0.42f, 0));
```

Comment:
It should be 0.42f according to Minecraft (Java) code, but it doesn't feel right in game (seems too high). Are motion values in Bedrock different? The translation for entity motion in Geyser doesn't modify it. Does anyone have any idea what's wrong?

Reply:
When I implemented fishing rod pulling, even though I used the same exact Java values, movement was still not the same. So it must be something in Bedrock.